pydash/utilities.py

```diff
@@ -680,6 +748,86 @@ def result(obj, key, default=None):
     return ret
 
 
+def stub_array():
```

Comment:
Since we're using `stub_dict` instead of `stub_object`, we should use `stub_list` instead of `stub_array`.